### PR TITLE
Feature wagercardupdate

### PR DESCRIPTION
**Pull request by AustinRobinsonDev**
Updates to wager card design and function

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
     "dotenv": "^10.0.0",
     "ethereum-waffle": "^3.3.0",
     "ethers": "^5.1.4",
+    "font-awesome": "^4.7.0",
```
Comment (AustinRobinsonDev) [inline package.json:23]: need to yarn remove font-awesome i forgot

File: src/components/WagerCard.js [hunk 2]
```diff
@@ -139,7 +139,7 @@ const WagerCard = (props) => {
           <Button fullWidth="true" onClick={alertWager}>
             Match <br /> Wager
           </Button>
-          <AlertDialog onClose={handleCloseAlert} userDisagrees={handleCloseAlert} userAgrees={onMatchWagerPressed} open={matchPressed}/>
+          <AlertDialog onClose={handleCloseAlert} ethAmount={props.amount} userDisagrees={handleCloseAlert} userAgrees={onMatchWagerPressed} open={matchPressed}/>
```
Comment (AustinRobinsonDev) [inline src/components/WagerCard.js]: need to remove that prop decided to go without

**Comment by JaredBorders** [comment]
Need to merge this into develop, not main

**Review by JaredBorders — CHANGES_REQUESTED** [review]
Changes look great! Just need to change the branch we are merging into to develop

**Pull request merged**